ci/resources/pgbouncer/pgbouncer_stimulus.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+echo $PWD
+PGPASSWORD=datadog embedded/pg_REL9_4_0/bin/psql -h localhost -p 15433 -U datadog -w -c "SELECT * FROM persons" datadog_test
```

Comment:
Having this hardcoded path here isn't right because it depends on the postgres version.